### PR TITLE
docs(readme): fix some grammar and some error information, and add other language links in the readme

### PR DESCRIPTION
**Pull request by XdpCs**
fix some grammar and some error information, and add other language links in the readme

### CODE REVIEW

**Comment by XdpCs** [comment]
Why did you close this pull request?

**Comment by a8m** [comment]
> Why did you close this pull request?

Accidentally, but there are unnecessary/incorrect changes to the README file in this PR.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## ent - An Entity Framework For Go
+# ent - An Entity Framework For Go
```
Comment (a8m) [inline README.md:1]: Please revert.

Comment (XdpCs) [reply README.md:1]: [MD041 - First line in a file should be a top-level heading](https://github.com/DavidAnson/markdownlint/blob/v0.31.1/doc/md041.md#md041---first-line-in-a-file-should-be-a-top-level-heading)

File: README.md [hunk 2]
```diff
@@ -20,38 +20,40 @@ with large data-models.
...
+
+The documentation for developing and using ent is available at: [https://entgo.io]([https://entgo.io).
 
 For discussion and support, [open an issue](https://github.com/ent/ent/issues/new/choose) or join our [channel](https://gophers.slack.com/archives/C01FMSQDT53) in the gophers Slack.
 
 ## Join the ent Community
-Building `ent` would not have been possible without the collective work of our entire community. We maintain a [contributors page](doc/md/contributors.md)
-which lists the contributors to this `ent`. 
+
+Building `ent` would not have been possible without the collective work of our entire community. We maintain a [contributors page](doc/md/contributors.md).
```
Comment (a8m) [inline README.md:38]: The extra dot in not correct. Markdown single line breaks are not translated to HTML line breaks.

File: README.md (hunk 1 above)
Comment (XdpCs) [reply README.md:1]: If you don't approve this standard, I will revert it.

File: README.md (hunk 2 above)
Comment (XdpCs) [reply README.md:38]: OK. I will fix it.

**Comment by XdpCs** [comment]
> > Why did you close this pull request?
> 
> Accidentally, but there are unnecessary/incorrect changes to the README file in this PR.

I am a rookie. When I study the ORM framework, I read the README. There are some markdown warnings. I want to receive your feedback on this process and learn from it.

**Pull request closed without merging**